Mention deno in docs (#1736)
sublimelsp/LSP: docs/src/language_servers.md

## TypeScript / JavaScript

- Node.js: Follow installation instructions on [LSP-typescript](https://github.com/sublimelsp/LSP-typescript).

> - deno.land: Follow installation instructions on [LSP-Deno](https://github.com/sublimelsp/LSP-Deno).

Reviewer: I believe it's called `Deno`. The `deno.land` is just a domain of the website.

> - Node.js: Follow installation instructions on [LSP-typescript](https://github.com/sublimelsp/LSP-typescript).
> - deno.land: Follow installation instructions on [LSP-Deno](https://github.com/sublimelsp/LSP-Deno).

Reviewer: Saying "Node.js" is maybe too narrow because the server can be used for plain JS without the Node server also. Could reformat it similar to:

```suggestion
Follow installation instructions on [LSP-typescript](https://github.com/sublimelsp/LSP-typescript).

For development with the Deno framework follow installation instructions on [LSP-Deno](https://github.com/sublimelsp/LSP-Deno).
```
